robottelo/hosts.py

```diff
@@ -1392,15 +1394,25 @@ def is_upstream(self):
         :return: True if no downstream satellite RPMS are installed
         :rtype: bool
         """
-        return self.execute('rpm -q satellite-capsule &>/dev/null').status != 0
+        return self.execute(f'rpm -q {self.product_rpm_name}').status != 0
+
+    @cached_property
+    def is_stream(self):
+        """Check if the Capsule is a stream release or not
+
+        :return: True if the Capsule is a stream release
+        :rtype: bool
+        """
+        if self.is_upstream:
+            return False
+        return (
+            'stream' in self.execute(f'rpm -q --qf "%{{RELEASE}}" {self.product_rpm_name}').stdout
```

Comment:
I'd suggest to not query rpm with only %{RELEASE} filter, instead query entire rpm string if stream is present, it'll verify that stream isn't in either VERSION, or RELEASE or NAME? non-blocking
```suggestion
            'stream' in self.execute(f'rpm -q --qf %{{RELEASE}} {self.product_rpm_name}').stdout
```

Reply:
I have discussed it with @evgeni in [the chat](https://redhat-internal.slack.com/archives/C04LA8ALR5Z/p1686735736238319).
The `stream` flag will be present only in `RELEASE`. It is nonsense to put it in `NAME` or `VERSION`. @evgeni could correct me here.

Reply:
Yeah, the way it's setup today, it will be in `%{release}` or nowhere at all.